cmd/link/userrole/cmd.go

```diff
@@ -124,6 +137,12 @@ func run(cmd *cobra.Command, argv []string) (err error) {
 
 	err = ocmClient.LinkAccountRole(currentAccount.ID(), roleArn)
 	if err != nil {
+		if errors.GetType(err) == errors.Forbidden {
+			reporter.Errorf("Only organization admin can run this command. "+
+				"Please ask someone with the organization admin role to run the following command \n\n"+
+				"\t rosa link user-role --role-arn %s --account-id %s", roleArn, currentAccount.ID())
+			return err
+		}
```

Comment:
Since this is not really the case at the moment, let's remove this message and just error out directly with `reporter.Errorf("%s", err)`